symfony/plugins/orangehrmAdminPlugin/test/Service/LocationServiceTest.php

```diff
@@ -44,6 +50,18 @@ protected function setUp(): void
         $this->locationService = new LocationService();
         $this->fixture = Config::get(Config::PLUGINS_DIR) . '/orangehrmAdminPlugin/test/fixtures/LocationDao.yml';
         TestDataService::populate($this->fixture);
+        $this->getContainer()->register(
```

Comment:
refer this `symfony/plugins/orangehrmCorePlugin/test/Service/HomePageServiceTest.php` to how to write KernelTestCase. so you can remove `use ServiceContainerTrait;`

Reply:
Done